Add feature gate StorageNamespaceIndex for v1.32 (#49359)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates/StorageNamespaceIndex.md

---
title: StorageNamespaceIndex
content_type: feature_gate

_build:
  list: never
  render: false

stages:
  - stage: beta 
    defaultValue: true
    fromVersion: "1.30"
---

> Enabling this gate allows namespace indexer for namespace scoped resources

Reviewer: ```suggestion
Enable a namespace indexer for namespace scoped resources
```
